scope.go

```diff
@@ -449,7 +460,8 @@ func (s *scope) Snapshot() Snapshot {
 		ss.tm.RLock()
 		for key, t := range ss.timers {
 			name := ss.fullyQualifiedName(key)
-			snap.timers[name] = &timerSnapshot{
+			id := ss.uniqueID(name, tags, buf)
```

Comment:
Hey instead of adding a new method `uniqueID` its probably best to use the already existing method `func KeyForPrefixedStringMap(prefix string, stringMap map[string]string) string` which resides in `key_gen.go`.